### PR TITLE
remove testZip step for selendroid

### PR DESCRIPTION
**Pull request by aluedeke**
fix for appium/appium#4414

### CODE REVIEW

**Comment by aluedeke** [comment]
appium/appium#4541

**Comment by bootstraponline** [comment]
:+1:

File: lib/helpers.js [hunk 1]
```diff
@@ -11,64 +11,25 @@ var logger = require('./logger'),
...
-            logger.error("Stdout: " + stdout);
-            cb("Archive could not be unzipped, check appium logs.", null);
-          }
-        });
-      }
+    if (exports.isWindows()) {
+      var zip = new AdmZip(zipPath);
+      zip.extractAllTo(path.dirname(zipPath), true);
+      logger.debug("Unzip successful");
+      cb(null, null);
```
Comment (jlipps) [inline lib/helpers.js]: can we take the opportunity to fix this pattern? this should just be `cb();`

File: lib/helpers.js [hunk 2]
```diff
@@ -11,64 +11,25 @@ var logger = require('./logger'),
...
+          logger.debug("Unzip successful");
+          cb(null, stderr);
         } else {
-          logger.error("Zip file " + zipPath + " was not valid");
+          logger.error("Unzip threw error " + err);
           logger.error("Stderr: " + stderr);
           logger.error("Stdout: " + stdout);
-          cb("Zip archive did not test successfully, check appium server logs " +
-             "for output", false);
+          cb("Archive could not be unzipped, check appium logs.", null);
```
Comment (jlipps) [inline lib/helpers.js]: can we take the opportunity to fix this? we should be calling back with an error object, e.g., `cb(new Error("Archive could not be unzipped, check appium logs."))`, and we don't want to pass another param if we have an error.

**Comment by jlipps** [comment]
made a few comments, then good to go

**Comment by paymand** [comment]
The latest unzip version on Ubuntu (6.0-12ubuntu1.3) has the fix and testZip step doesn't fail anymore.

**Pull request closed without merging**